bindings/java/src/main/com/apple/foundationdb/EventKeeper.java

```diff
@@ -167,6 +167,72 @@ default long getTime(Event event, TimeUnit theUnit) {
 			public boolean isTimeEvent() {
 				return true;
 			}
-		};
+		},
+		/**
+		 * The number of single-key get requests.
+		 */
+		GET_REQUESTS,
+		/**
+		 * The number of getKey requests.
+		 */
+		GET_KEY_REQUESTS,
+		/**
+		 * The number of requests for an estimated range size(i.e. {@link
+		 * FDBTransaction#getEstimatedRangeSizeBytes(Range)} or {@link FDBTransaction#getEstimatedRangeSizeBytes(byte[],
+		 * byte[])})
+		 */
+		ESTIMATED_RANGE_SIZE_REQUESTS,
+		/**
+		 * The number of requests for a range split point.
+		 */
+		GET_RANGE_SPLIT_POINT_REQUESTS,
+		/**
+		 * The number of calls to add a conflict range
+		 */
+		ADD_CONFLICT_RANGES,
+		/**
+		 * The number of set calls.
+		 */
+		SETS,
+		/**
+		 * The number of clear requests made.
+		 */
+		CLEAR_REQUESTS,
+		/**
+		 * The number of mutation requests made.
+		 */
+		MUTATIONS,
+		/**
+		 * The number of setOption requests made.
+		 */
+		SET_TRANSACTION_OPTION,
+		/**
+		 * The number of commits requested.
+		 */
+		COMMITS,
```

Comment:
nit: this could arguably include the `_REQUESTS` suffix since it can result in a request being made to the cluster. If you had some other rationale for when you used this suffix, that's fine too.

```suggestion
		COMMIT_REQUESTS,
```